controllers/controllers.js

```diff
@@ -0,0 +1,28 @@
+const {fetchTopics, fetchArticlesById} = require('../models/models'); 
+
+const getTopics = (req,res) => {
+    fetchTopics()
+    .then((topics) => {
+        res.status(200).send(topics)
+    })
+    .catch((err) => {
+        console.log(err)
+    })
+}
+
+const getArticlesById = (req, res, next) => {
+    const { article_id: articleId } = req.params;
+    fetchArticlesById(articleId)
+    .then((article) => {
+        res.status(200).send({article})
+    })
+    .catch((err) => {
+        console.log(err)
+        next(err)
+    })
+};
+
```

Comment:
Not a major concern at this stage - but this controllers file could get rather hefty rather quickly. Consider separate files for controllers responsible for different resources - topics, articles, comments etc.

Reply:
done!